sources/packages/api/src/services/auth-service/auth-service.service.ts

```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@nestjs/common';
+import jwtDecode from 'jwt-decode';
+import { UserInfo } from '../../types';
+
+@Injectable()
+export class AuthService {
+  parseAuthorizationHeader(token: string): UserInfo {
+    return jwtDecode(token) as UserInfo;
```

Comment:
Please create the tech debit to work on the authentication gaps.

Reply:
Yes, two technical debt.